code/controllers/subsystems/garbage.dm

```diff
@@ -269,12 +308,9 @@ SUBSYSTEM_DEF(garbage)
 
 
 	if(isnull(D.gc_destroyed))
-		if (SEND_SIGNAL(D, COMSIG_PARENT_PREQDELETED, force)) // Give the components a chance to prevent their parent from being deleted
-			return
 		D.gc_destroyed = GC_CURRENTLY_BEING_QDELETED
 		var/start_time = world.time
 		var/start_tick = world.tick_usage
-		SEND_SIGNAL(D, COMSIG_PARENT_QDELETING, force) // Let the (remaining) components know about the result of Destroy
```

Comment:
do not remove this wtf, this is used for dereferencing components or objects

Reply:
That comsig doesn't do anything.

Reply:
Whoever ported it did half a job and had a full smile.